index.html

```diff
@@ -33,9 +33,149 @@
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+           </div>
+       
+        <div class="grid-item third">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fourth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fiveth"><div class="item image"></div> 
+        <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+        <div class="tagcontainer">
+         <label>Ruby on Rails</label>
+         <label>Javascript</label>
+         <label>CSS</label>
+         <label>HTML</label>
+        </div>
+        <button class="see-post"><p class="see-button">see project</p></button></div>
+        <div class="grid-item sixth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item ">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           <div class="tagcontainer">
+            <label>Ruby on Rails</label>
+            <label>Javascript</label>
+            <label>CSS</label>
+            <label>HTML</label>
+           </div>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+    <div class="grid-container-another">
+        <div class="about-me">
+            <h2 class="about">About Me</h2>
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+                <button class="get-my-resume"><p class="resume">Get My Resume</p></button>
```

Comment:
- [ ] The button for the about me section is also not the same as the figma template kindly fix this as well :grin:
![btn-about](https://user-images.githubusercontent.com/89055502/235299918-e9b89c28-6060-4d8a-a6e0-5fbca717c079.png)